pkg/services/publicdashboards/database/database.go

```diff
@@ -45,7 +45,7 @@ func (d *PublicDashboardStoreImpl) ListPublicDashboards(ctx context.Context, org
 			Join("LEFT", "dashboard", "dashboard.uid = dashboard_public.dashboard_uid AND dashboard.org_id = dashboard_public.org_id").
 			Cols("dashboard_public.uid", "dashboard_public.access_token", "dashboard_public.dashboard_uid", "dashboard_public.is_enabled", "dashboard.title").
 			Where("dashboard_public.org_id = ?", orgId).
-			OrderBy("is_enabled DESC, dashboard.title ASC")
+			OrderBy("dashboard.title IS NULL ASC, is_enabled DESC")
```

Comment:
@jalevin I tried this and it works but didn't test it inside Grafana with different DBs. WDYT?
This syntax is supported by the 3 DBMS (Postgres, MySQL and SQLite)

Reply:
sorted on NULL is inverted on postgres vs. mysql/sqlite. I didn't see an easy way to invert this. Does this "IS NULL ASC" handle that?!

Reply:
I'm all for it if it works!!

Reply:
once https://github.com/grafana/grafana-partnerships-team/issues/421 is merged we won't have to worry about this anyways, right?

Reply:
I would like to have this list always sorted but after public dashboard deletion support this orphan pubdash case should not be that common.

Reply:
This was what I spent WAY too long trying to figure out. Nice work.